docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -129,13 +140,52 @@ For an end user, the UX to update the kubernetes version of the control plane an
 As an end user, I want to be able to easily scale up/down the number of replicas for each set of worker nodes in the cluster.
 - Currently, (for a cluster with 3 machine deployments) this is possible by updating these three different objects representing the sets of worker nodes in the pool. An easier user experience would be to update a single object to enable the scaling of multiple sets of worker nodes.
 
+#### Story 4 - Use ClusterClass to easily modify Clusters
+As an end user, I want to be able to easily change the configuration of all Clusters of a `ClusterClass`. For example, I want to be able to change the ControlPlane configuration by  
```

Comment:
can we be more specific about "end user", would it be a cluster admin in this scenario?

Reply:
Looking at some of our other proposals, we seem to be calling this persona "cluster operator". I used "end user" because the 3 pre-existing user stories used it. I"m fine with consistently changing it to "cluster operator"

Reply:
Done (I'll push later)